### PR TITLE
[ACA-1911] Un-Share file - permission error notification

### PR DESCRIPTION
**Pull request by pionnegru**
## PR Checklist
Please check if your PR fulfills the following requirements:

```
- [ ] The commit message follows our guidelines: https://github.com/Alfresco/alfresco-content-app/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
```

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Application / Infrastructure changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/assets/i18n/en.json [hunk 1]
```diff
@@ -265,6 +265,9 @@
             "INHERITED_PERMISSIONS_BUTTON": "Permission Inherited"
         }
     },
+    "SHARED_LINK": {
+        "UNSHARE_PERMISSION_ERROR": "You dont have permission to unshare this file"
```
Comment (DenysVuika) [inline src/assets/i18n/en.json]: Please raise a ticket to LOC team regarding extra i18n files needed

**Comment by gbroadbent** [comment]
Thank you, Denys.  Yes, please create a LOC ticket and attach the EN and Localised files so we can update all languages accordingly.
Please add the relevant apostrophe to: You dont have permission to unshare this file
To read:

You don't have permission to unshare this file

Best regards,

Gloria

File: src/assets/i18n/en.json (hunk 1 above)
Comment (DenysVuika) [inline src/assets/i18n/en.json]:
```suggestion
        "UNSHARE_PERMISSION_ERROR": "You don't have permission to unshare this file"
```

**Comment by gbroadbent** [comment]
Hello Denys,

Regarding this email below, has anyone raised a LOC ticket with the EN file, so we can update the languages?

Thank you!

Best regards,
Anni

Anni Nygren
Localisation Specialist
Tel: +44 1628 876517 | Skype: anni.nygren

[cid:image001.png@01D25B73.AC327400]



From: Gloria Camino <gloria.camino@alfresco.com<mailto:gloria.camino@alfresco.com>>
Date: Saturday, 20 October 2018 at 10:49
To: Alfresco/alfresco-content-app <reply@reply.github.com<mailto:reply@reply.github.com>>
Cc: Subscribed <subscribed@noreply.github.com<mailto:subscribed@noreply.github.com>>, Anni Nygren <anni.nygren@alfresco.com<mailto:anni.nygren@alfresco.com>>
Subject: Re: [Alfresco/alfresco-content-app] [ACA-1911] Un-Share file - permission error notification (#743)

Thank you, Denys.  Yes, please create a LOC ticket and attach the EN and Localised files so we can update all languages accordingly.
Please add the relevant apostrophe to: You dont have permission to unshare this file
To read:

You don't have permission to unshare this file

Best regards,

Gloria

**Pull request merged**